src/test/java/core/basesyntax/HelloWorldTest1.java

```diff
@@ -0,0 +1,183 @@
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class HelloWorldTest1 {
+    private static RegistrationService registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void userWithNullLogin_NotOk() {
+        User user = new User();
+        user.setLogin(null);
+        user.setPassword("12345678");
+        user.setAge(22);
+        assertThrows(CustomeException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void nullPassword_NotOk() {
+        User user = new User();
+        user.setLogin("user12345");
+        user.setPassword(null);
+        user.setAge(41);
+        assertThrows(CustomeException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void nullAge_NotOk() {
+        User user = new User();
+        user.setAge(null);
+        Exception exception = assertThrows(CustomeException.class, () ->
+                registrationService.register(user));
+        String excepted = "User login,password or age is null";
+        String actual = exception.getMessage();
+        assertTrue(actual.contains(excepted));
+    }
+
+    @Test
+    void threeLettersLogin_NotOk() {
+        User user = new User();
+        user.setLogin("abc");
+        user.setPassword("abrakadabra");
+        user.setAge(83);
+        Exception exception = assertThrows(CustomeException.class, () ->
+                registrationService.register(user));
+        String excepted = "User login is need to be at least 6 letters!";
+        String actual = exception.getMessage();
+        assertTrue(actual.contains(excepted));
+    }
+
+    @Test
+    void fiveLettersLogin_NotOk() {
+        User user = new User();
+        user.setLogin("abcdf");
+        user.setPassword("messagetotheworld");
+        user.setAge(41);
+        Exception exception = assertThrows(CustomeException.class, () ->
+                registrationService.register(user));
+        String excepted = "User login is need to be at least 6 letters!";
+        String actual = exception.getMessage();
+        assertTrue(actual.contains(excepted));
+    }
+
+    @Test
+    void eightLettersLogin_Ok() {
+        User user = new User();
+        user.setLogin("Vitaliks");
+        user.setAge(22);
+        user.setPassword("normalpassword321");
+        registrationService.register(user);
+        assertTrue("Vitaliks".equals(user.getLogin()));
+    }
+
+    @Test
+    void emptyloginUser_NotOk() {
+        User user = new User();
+        user.setLogin("");
+        user.setAge(32);
+        user.setPassword("thebestpassword");
+        Exception exception = assertThrows(CustomeException.class, () ->
+                registrationService.register(user));
+        String excepted = "User login or password is empty";
```

Comment:
The expected message 'User login or password is empty' does not match the actual message format. Ensure the expected message matches the actual exception message.